frictionless/lbnlisoqueues.py

```diff
@@ -0,0 +1,46 @@
+"""Provide datapackage details specific to the EIP Infrastructure archives."""
+
+from . import core
+from . import contributors
+from . import licenses
+
+lbnlisoqueues = {
+    "name": "raw-lbnlisoqueues",
+    "title": "Raw LBLN ISO Interconnection Queues",
```

Comment:
Is it LBL (Lawrence Berkeley Lab) or LBNL (Lawrence Berkeley National Lab). I think these may be different things. And in any case LBLN seems like it's probably a typo

Reply:
Their domain and website give mixed signals but I'm pretty sure it is [Lawrence Berkeley National Lab](https://en.wikipedia.org/wiki/Lawrence_Berkeley_National_Laboratory).